wp-content/themes/core/components/blocks/lead_form/Lead_Form_Block_Controller.php

```diff
@@ -255,6 +244,8 @@ public function get_form(): string {
 			return '';
 		}
 
-		return gravity_form( $this->form, false, false, false, null, false, 1, false );
+		$allowed_blocks = [ 'gravityforms/form' ];
```

Comment:
Should this be a method and or filterable JIC we need to add additional blocks?

Reply:
Great question, at some point yes, I think that's correct. For now or at least until we need it, I think this is ok as we only tend to use GF.

Reply:
Noted as a future TODO